app/models/user.rb

```diff
@@ -5,4 +5,13 @@ class User < ActiveRecord::Base
          :recoverable, :rememberable, :trackable, :validatable
 
   has_many :comments
+  has_many :requests
+
+  def requested_book?(book)
```

Comment:
Better if you name it `requested?(book)`. Since book is an argument you have:
`user.requested? book` and you get the idea when you call it.